stylesheets/main.css

```diff
@@ -44,6 +45,8 @@
     --light-purple:#e879f9;
     --red-color:#ef4444;
     --medium-green:#1a9b49;
+    --ex-light-red: #fecaca;
+    --ex-light-purple:#efd6f3;
 
 }
```

Comment:
Use of variable names as primary colour etc is suggested, so even if you decide to change the colour in future, you do not need to change the names from multiple places.

Reply:
I know that thing but before that class, I was already done with 3 features that's why I continue like this. I will keep in mind this for the next project